configs/opencv-ovms/scripts/grpc_python.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import sys
+sys.path.append("/model_server/demos/common/python")
+
+import argparse
+import datetime
+import cv2
+import grpc
+from client_utils import print_statistics
+from tritonclient.grpc import service_pb2, service_pb2_grpc
+from grpc_postprocess import *
+
+def openInputSrc(input_src):
+    # OpenCV RTSP Stream
+    stream = cv2.VideoCapture(input_src)
+    if not stream.isOpened():
+        print('Unable to open source:' + input_src)
+        exit(-1)
+    return stream
+
+def setupGRPC(address, port):
+    address = "{}:{}".format(address, port)
+    # Create gRPC stub for communicating with the server
+    channel = grpc.insecure_channel(address)
+    grpc_stub = service_pb2_grpc.GRPCInferenceServiceStub(channel)
+    return grpc_stub
+
+def getModelSize(model_name):
```

Comment:
is this temporary? I thought the plan was to get the model size from the ovms config file....